### PR TITLE
P0 - Login and Signup

### PR DESCRIPTION
**Pull request by rishimish**
Created login and Signup

live link : https://deploy-preview-2--bee-you.netlify.app/pages/login.html

Screenshot 2022-02-20 at 4 47 03 PM

Screenshot 2022-02-20 at 4 47 28 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,87 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Black Unlimited</title>
```
Comment (poojavishnoi) [inline index.html:7]: Add meta tag for author and description.

**Review by poojavishnoi — APPROVED** [review]
Looks good.

**Review by zabihshaik — APPROVED** [review]
Everything looks good.
Better to use meta tags for SEO.
UI looks Great.

File: index.html (hunk 1 above)
Comment (rishimish) [reply index.html:12]: thanks, added that.

**Comment by rishimish** [comment]
> Everything looks good. Better to use meta tags for SEO. UI looks Great.

thanks, added that.

**Pull request merged**